.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Extractors/DateTimeExtractorChs.cs

```diff
@@ -65,7 +61,7 @@ public List<Token> MergeDateAndTime(string text)
                 return ret;
             }
 
-            ers.Sort(delegate(ExtractResult er1, ExtractResult er2)
+            ers.Sort(delegate (ExtractResult er1, ExtractResult er2)

```

Comment:
This will conflict with the Sort change in PR #92.

Reply:
Updated PR with master branch merge